### PR TITLE
fix(medcryptor): delete all chats in space throws err

### PR DESCRIPTION
**Pull request by alexandra-marin**
#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/9163/mc-desktop-deleting-rooms-in-patient-space-causes-error
#### Testing instructions  
1 create a patient space
2 delete the auto generated rooms
3 goes back to all chats

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: src/whitelabel/helpers/space.js [hunk 1]
```diff
@@ -3,9 +3,14 @@
     e.g. finding the current space ID, or determining whether the current room is Patient or Internal
 */
 
+const { computed } = require('mobx');
 const { chatStore, contactStore, User } = require('peerio-icebear');
 
 class SPACE {
+    @computed get currentSpaceDeleted() {
```
Comment (anri-asaturov) [inline src/whitelabel/helpers/space.js]: This getter doesn't derive (compute) any values (negation is too cheap), doesn't need to be computed.

**Comment by ebilg** [comment]
when I create a new patient space, then select a room, I get a blue screen

**Comment by alexandra-marin** [comment]
@ebilg have merged with dev, please try again and send me the logs if it still happens

**Comment by ebilg** [comment]
can be merged

**Pull request merged**